charts/camunda-platform-alpha/values.yaml

```diff
@@ -2276,16 +2295,26 @@ identityPostgresql:
 identityKeycloak:
   ## @param identityKeycloak.enabled Enable Identity Keycloak Helm chart. It is used incorporate with "global.identity.keycloak" to use your own Keycloak instead of the one comes with Camunda Helm chart
   enabled: true
+  global:
+    ## Compatibility adaptations for Kubernetes platforms
+    ##
+    compatibility:
+      ## Compatibility adaptations for Openshift
+      ##
+      openshift:
+        ## @param identityKeycloak.global.compatibility.openshift.adaptSecurityContext Adapt the securityContext sections of the deployment to make them compatible with Openshift restricted-v2 SCC: remove runAsUser, runAsGroup and fsGroup and let the platform use their allowed default IDs. Possible values: force (perform the adaptation always), disabled (do not perform adaptation)
+        ##
+        adaptSecurityContext: "{{ .Values.global.compatibility.openshift.adaptSecurityContext | default \"disabled\" }}"
   ## @param identityKeycloak.nameOverride the name used for Keycloak.
   nameOverride: "keycloak"
 
   ## @extra identityKeycloak.image configuration.
   # https://hub.docker.com/r/bitnami/keycloak/tags
   image:
     ## @param identityKeycloak.image.repository image repo
-    repository: bitnami/keycloak
+    repository: camunda/keycloak
     ## @param identityKeycloak.image.tag image tag
-    tag: 23.0.7
+    tag: 23.0.7-debian-12-r5
```

Comment:
@aabouzaid I hope I used the correct tag for keycloak

Reply:
@hamza-m-masood We can use it like that now but please ask the InfraEx team to provide a version-only tag (e.g. `23.0.7` which I cannot find in the [Docker hub repo](https://hub.docker.com/r/camunda/keycloak/tags))

Reply:
I asked infraex team if they can add the tag `23.0.7` to the list of tags we support.

Reply:
I have added the `23.0.7` image to the helm chart. I have also tested it and it is working.